scripts/cmake/DocumentationSetup.cmake

```diff
@@ -38,4 +38,36 @@ if(DOXYGEN_FOUND)
 
     configure_file(Documentation/Doxyfile.in ${PROJECT_BINARY_DIR}/Doxyfile)
 
+    if (NOT BASH_TOOL_PATH STREQUAL "BASH_TOOL_PATH-NOTFOUND"
+            AND NOT PYTHON_EXECUTABLE STREQUAL "PYTHON_EXECUTABLE-NOTFOUND")
```

Comment:
The string comparison is not necessary. Simply check like this:

```cmake
if (NOT BASH_TOOL_PATH AND NOT PYTHON_EXECUTABLE)
```

From the docs:

> False if the constant is 0, ... or ends in the suffix -NOTFOUND.